languages/ruby/spec/oso/polar/data_filtering/helpers.rb

```diff
@@ -14,6 +14,20 @@ def check_authz(actor, action, resource, expected)
     end
   end
 
+  def check_authzd(actor, action, resource, expected)
```

Comment:
This difference between this and `check_authz` is very unclear from the name.

we can address as part of renaming `authzd resources`